databricks/koalas/indexes.py

```diff
@@ -2981,6 +2982,39 @@ def item(self):
         """
         return self._kdf.head(2)._to_internal_pandas().index.item()
 
+    def equal_levels(self, other):
+        """
+        Return True if the levels of both MultiIndex objects are the same
+
+        Examples
+        --------
+        >>> from databricks.koalas.config import set_option, reset_option
+        >>> set_option("compute.ops_on_diff_frames", True)
+
+        >>> kmidx1 = ks.MultiIndex.from_tuples([("a", "x"), ("b", "y"), ("c", "z")])
+        >>> kmidx2 = ks.MultiIndex.from_tuples([("b", "y"), ("a", "x"), ("c", "z")])
+        >>> kmidx1.equal_levels(kmidx2)
+        True
+
+        >>> kmidx2 = ks.MultiIndex.from_tuples([("a", "x"), ("b", "y"), ("c", "j")])
+        >>> kmidx1.equal_levels(kmidx2)
+        False
+
+        >>> reset_option("compute.ops_on_diff_frames")
+        """
+        nlevels = self.nlevels
+        if (nlevels != other.nlevels) or (len(self) != len(other)):
```

Comment:
Is it possible to avoid `len()` which runs a Spark job?

Reply:
yeah, I think we can just don't check here.